### PR TITLE
Allow "=" in dc

### PR DESCRIPTION
**Pull request by kumar35959**
Allow "=" in dc(Domain Component)

After this commit is merged, https://github.com/foxpass/foxpass/blob/master/requirements/production.txt will be updated with latest commit id for divvy-client-python module

### CODE REVIEW

File: divvy/protocol.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 
 class Translator(object):
-    STRING_REGEXP = re.compile('^[^"=\\s]+$')
+    STRING_REGEXP = re.compile('^[^"\\s]+$')
```
[the remark this answers is not in the document]
Comment (kumar35959) [reply divvy/protocol.py]: To clarify, you are suggesting to use the following

`STRING_REGEXP = re.compile(r'^[^"]+$')`

[the remark this answers is not in the document]
Comment (aren) [reply divvy/protocol.py]: No. See the recommended regex here: https://github.com/button/divvy.

```
Strings must conform to the following regexes:

Unquoted: /^[^"=\s]+$/
Quoted: /^"[^"\n]*"$/
```

[the remark this answers is not in the document]
Comment (aren) [reply divvy/protocol.py]: Spaces are allowed, but newlines are not.

[the remark this answers is not in the document]
Comment (kumar35959) [reply divvy/protocol.py]: As per the discussions, new requirements are 

To consolidate all your comments
- Use raw string (`r`)
- Allow `spaces`
- Allow `=`
- Do not allow `'\n'`

In divvy-client, we receive `unquoted string`
`('33: v: ', u'stagingdc=.rtcdndc=.caffeinedc=.tvdc=')`

New regex to satisfy all the above requirements is 

`STRING_REGEXP = re.compile(r'^[^"\n]+$')`

[the remark this answers is not in the document]
Comment (kumar35959) [reply divvy/protocol.py]: Done. Taken care of new requirements

**Pull request merged**